formula/src/main/java/com/instacart/formula/FormulaContext.kt

```diff
@@ -93,39 +93,37 @@ abstract class FormulaContext<out Input, State> internal constructor(
         return child(formula, Unit)
     }
 
+    /**
+     * Returns the latest output of the [child] formula. Formula runtime ensures the [child]
+     * is running, manages its internal state and will trigger `evaluate` if needed.
+     */
```

Comment:
Should we mention different error handling here, as compared to `childOrNull()`?